atomic_reactor/plugins/pre_koji_parent.py

```diff
@@ -229,6 +230,16 @@ def manifest_list_entries_match(self, image, build_id):
         if koji_archives_data == manifest_list_data:
             self.log.info('Deeper manifest list check verified v2 manifest references match')
             return True
+
+        # Customer facing registry might contain additional v2 digests in manifest lists
```

Comment:
This had nothing with registries. Any tag can contain more arches we care about. API just allows that